home.html

```diff
@@ -0,0 +1,183 @@
+            <a href="" class="desktopUl-li upperCase">sponsors</a>
+            <a href="" class="desktopUl-li upperCase">Persons</a>
+            <a href="" class="desktopUl-li upperCase">contact us</a>
+        </div>
+    </div>
+    </header>
+    <main id="main">
+        <section class="mainInfos">
+            <h1 class="title upperCase">"Hello,Date Lovers"</h1>
+            <h2 class="title-2 upperCase">Dates Festival 2021
+               Mauritania Tagant.</h2>
+               <p class="mainPara">
+                Dates are the fruit of the date palm tree, which is grown in tropical regions of the world.
+                Dates are healthy and delicious and have become very popular recently.
+               </p>
+               <div class="mainDate">
+                <h3 class="eventDate">2021.12.23(FRI) ~ 25(SUN)</h3>
+                <h4 class="eventPlace">@ National Dates Festival of Mauritania, Tagant.</h4>
+            </div>
+        </section>
+        <section class="mainProgram">
+            <h5 class="programTitle capitalize">main program</h5>
+            <div class="separator"></div>
+            <ul class="programContainer">
+                <li class="programli"><img class="mainProgramImg" src="images/camelsImage.jpeg" alt="camels">
+                    <h1 class="mPImgtitle">Camels</h1>
+                    <p class="mPImgPara">Enjoy riding camels in the desert and 
+                        drinking camel milk.
+                    </p>
+                </li>
+                <li class="programli"><img class="mainProgramImg" src="images/yellowDates.jpeg" alt="date">
+                    <h2 class="mPImgtitle">Yellow Dates</h2>
+                    <p class="mPImgPara">The real taste of fruit, this is the date befor becoming darker.
+                    </p>
+                </li>
+                <li class="programli"><img class="mainProgramImg" src="images/poetryImage.jpg" alt="poetryImage">
+                    <h2 class="mPImgtitle">Poetry</h2>
+                    <p class="mPImgPara">Listen to the best poets of the country and exclusive new verses.
+                    </p>
+                </li>
+                <li class="programli"><img class="mainProgramImg" src="images/Amnagel.jpeg" alt="Amnagel">
+                    <h2 class="mPImgtitle">Amnagal</h2>
+                    <p class="mPImgPara">Amnagel is the name of the colorful dates,which are also available.
+                    </p>
+                </li>
+                <li class="programli"><img class="mainProgramImg" src="images/blackDates.jpg" alt="">
+                    <h2 class="mPImgtitle">Black Dates</h2>
+                    <p class="mPImgPara">The classic one enjoy the taste of the fresh black dates.
+                    </p>
+                </li>
+            </ul>
+            <div class="programli pL-Button"><h1>Join Tagant Dates Festival 2021</h1></div>
+        </section>
+        <!-- start featured section for desktop -->
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.